drop-in/build.gradle

```diff
@@ -71,6 +71,7 @@ dependencies {
     api project(':bacs')
     api project(':wechatpay')
     api project(':voucher')
+    api project(':online-banking-pl')
```

Comment:
Alphabetical order

Reply:
> Did you test the app? It crashes when you click on Online Banking PL

what's the crash that you get? It doesn't crash with me!